<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QCustomPlot" name="widget" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>164</width>
          <height>138</height>
         </size>
        </property>
        <property name="title">
         <string>Options:</string>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioFilter">
           <property name="text">
            <string>No Filter</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioRunningAverage">
           <property name="text">
            <string>Running Average</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="radioSample">
           <property name="text">
            <string>Sample</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="radioKalman">
           <property name="text">
            <string>Kalman Filter</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer">
        <property name="maximumSize">
         <size>
          <width>43</width>
          <height>23</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="QFrame" name="frame">
        <property name="maximumSize">
         <size>
          <width>301</width>
          <height>164</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string># samples</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Sample Delay</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QSpinBox" name="spinBoxNumSample"/>
               </item>
               <item row="1" column="1">
                <widget class="QSpinBox" name="spinBoxSampleDelay"/>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QPushButton" name="pushButtonPauseResume">
                 <property name="text">
                  <string>Pause/Resume</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonRecord">
                 <property name="text">
                  <string>Record</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonSample">
               <property name="text">
                <string>Sample</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>groupBox</zorder>
   <zorder>widget</zorder>
   <zorder>pushButtonSample</zorder>
   <zorder>spinBoxNumSample</zorder>
   <zorder>spinBoxSampleDelay</zorder>
   <zorder>pushButtonPauseResume</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>pushButtonRecord</zorder>
   <zorder>verticalSpacer</zorder>
   <zorder>verticalSpacer_2</zorder>
   <zorder>frame</zorder>
   <zorder>horizontalSpacer</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1032</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInterface">
    <property name="title">
     <string>Sensor</string>
    </property>
    <addaction name="actionUS_Sensor"/>
    <addaction name="actionAccelerometer"/>
    <addaction name="actionGyroscope"/>
    <addaction name="actionGPS"/>
    <addaction name="actionCompass"/>
    <addaction name="actionAltimiter"/>
    <addaction name="actionIR_Sensor"/>
   </widget>
   <widget class="QMenu" name="menuInterface_2">
    <property name="title">
     <string>Interface</string>
    </property>
   </widget>
   <addaction name="menuInterface"/>
   <addaction name="menuInterface_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionBeaglebone">
   <property name="text">
    <string>Beaglebone</string>
   </property>
  </action>
  <action name="actionUS_Sensor">
   <property name="text">
    <string>US Sensor</string>
   </property>
  </action>
  <action name="actionAccelerometer">
   <property name="text">
    <string>Accelerometer</string>
   </property>
  </action>
  <action name="actionGyroscope">
   <property name="text">
    <string>Gyroscope</string>
   </property>
  </action>
  <action name="actionGPS">
   <property name="text">
    <string>GPS</string>
   </property>
  </action>
  <action name="actionCompass">
   <property name="text">
    <string>Compass</string>
   </property>
  </action>
  <action name="actionAltimiter">
   <property name="text">
    <string>Altimiter</string>
   </property>
  </action>
  <action name="actionIR_Sensor">
   <property name="text">
    <string>IR Sensor</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
